plugin/wavesurfer.regions.js

```diff
@@ -285,9 +285,15 @@ WaveSurfer.Region = {
         }
 
         if (this.element != null) {
+            // Calculate the left and width values of the region such that
+            // no gaps appear between regions.
+            var left = Math.round(this.start / dur * width);
+            var regionWidth =
+                Math.round(this.end / dur * width) - left;
+
             this.style(this.element, {
-                left: ~~(this.start / dur * width) + 'px',
-                width: ~~((this.end - this.start) / dur * width) + 'px',
```

Comment:
So your PR basically removes the `Math.floor` here. It may result in huge numbers like `25.4364564575467px`. Maybe let's just apply a `toFixed` so that the accuracy is just enough to avoid gaps? Like this:
```
left: (this.start / dur * width).toFixed(4) + 'px',
width: ~~((this.end - this.start) / dur * width).toFixed(4) + 'px',
```

I think 4 digits after comma should be enough.